deploy-help/configure.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash -e
+
```

Comment:
I think the `configure.sh` script is not necessary. It should be merged with `deploy.sh`.